src/Gaffer/FileSystemPath.cpp

```diff
@@ -258,7 +258,28 @@ bool FileSystemPath::isValid( const IECore::Canceller *canceller ) const
 
 	std::error_code e;
 
-	const std::filesystem::file_type t = std::filesystem::symlink_status( std::filesystem::path( this->string() ), e ).type();
+	std::filesystem::file_type t = std::filesystem::symlink_status( std::filesystem::path( this->string() ), e ).type();
+
+#if _MSC_VER < 1932
```

Comment:
What does this evaluate to on Linux? Is there any chance `_MSC_VER` ends up being interpreted as `0` because it's not defined?

Reply:
I didn't know that macro values evaluated to 0 if undefined, good to know. Fixed in https://github.com/GafferHQ/gaffer/pull/5492/commits/dd06cf54230dd62d5c675dc652ea5b862ae9933d